packages/libs/error-stack/Cargo.toml

```diff
@@ -17,21 +17,25 @@ tracing-error = { version = "0.2.0", optional = true }
 once_cell = { version = "1.10.0", optional = true }
 pin-project = { version = "1.0.10", optional = true }
 futures-core = { version = "0.3.21", optional = true, default-features = false }
+smallvec = { version = "1.9.0", optional = true }
 
 [dev-dependencies]
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
 futures = { version = "0.3.21", default-features = false, features = ["executor"] }
+expect-test = "1.3.0"
```

Comment:
Is it worth adding this as a dependency?

Reply:
I have been experimenting with the integration tests but then moved to a more simplistic test. Therefore I should remove this dependency.